### PR TITLE
Fix version error handling

### PR DESCRIPTION
**Pull request by chuckdumont**

### CODE REVIEW

**Comment by chuckdumont** [comment]
@ddumont

File: jaggr-core/WebContent/loaderExtCommon.js [hunk 1]
```diff
@@ -346,10 +346,14 @@ var params = {
 				addFoldedModuleName(dep, i, oFolded, oPrefixes);
 			}
 		}
-		return url + (argNames[0] === 'modules' ? ((url.indexOf("?") === -1 ? "?" : "&") + "count=" + i) : "" ) + 
-		             (sizeofObject(oFolded) ? ("&"+argNames[0]+"="+encodeURIComponent(encodeModules(oFolded))):"") + 
-		             (ids.length ? ("&"+argNames[1]+"=" + base64EncodeModuleIds(ids, base64Encoder, argNames[0] === 'modules' && hash)):"");
+		url +=(argNames[0] === 'modules' ? ((url.indexOf("?") === -1 ? "?" : "&") + "count=" + i) : "" ) +
```
Comment (ddumontatibm) [inline jaggr-core/WebContent/loaderExtCommon.js]: This looks fine logic-wise... but these nested and chained ternary expressions are not the easiest to follow...  

Do you think we can break these up to if statements to craft the URL?

File: jaggr-core/src/main/java/com/ibm/jaggr/core/impl/modulebuilder/javascript/JavaScriptModuleBuilder.java [hunk 2]
```diff
@@ -665,7 +669,9 @@ protected String moduleNameIdEncodingEndLayer(HttpServletRequest request, List<I
 			  .append(");"); //$NON-NLS-1$
 		}
 		// Finally, close the scoping function and invoke it.
-		sb.append("})();"); //$NON-NLS-1$
+		if (!noAMD) {
```
Comment (ddumontatibm) [inline jaggr-core/src/main/java/com/ibm/jaggr/core/impl/modulebuilder/javascript/JavaScriptModuleBuilder.java]: Should we put a comment here saying why we don't close the scope here if it **is** noAMD?
like...  will we not have a scope to close/invoke if noAMD is true?

**Comment by ddumontatibm** [comment]
otherwise lgtm

File: jaggr-core/WebContent/loaderExtCommon.js (hunk 1 above)
Comment (chuckdumont) [inline jaggr-core/WebContent/loaderExtCommon.js]: I did that with the if clause below.  I chose to leave well enough alone with the code that I didn't change.

File: jaggr-core/src/main/java/com/ibm/jaggr/core/impl/modulebuilder/javascript/JavaScriptModuleBuilder.java (hunk 2 above)
Comment (chuckdumont) [inline jaggr-core/src/main/java/com/ibm/jaggr/core/impl/modulebuilder/javascript/JavaScriptModuleBuilder.java]: Comment added

**Pull request merged**